mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,46 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.airqo.app">
 
     <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
@Mnoble-19 , somehow unrelated....in these manifests, isn't this where we are supposed to deal with the permissions issues we currently have where the app stays running in the background after a user has exited it?

Reply:
Thanks @Mnoble-19 ....so is this issue only in IOS?